src/SystemLoader_TEST.cc

```diff
@@ -30,29 +30,46 @@
 using namespace gz;
 using namespace sim;
 
+#ifdef _WIN32
+  constexpr char *kPluginDir = "bin";
+#else
+  constexpr char *kPluginDir = "lib";
+#endif
```

Comment:
I think these need to be additionally const-qualified.

```suggestion
#ifdef _WIN32
  constexpr const char *kPluginDir = "bin";
#else
  constexpr const char *kPluginDir = "lib";
#endif
```

Reply:
too fast :)

Reply:
ha ha, nice!